### ISSUE TITLE
Blixt will not scan BIP21 QR code with `lightning` invoice as first param

### ISSUE THREAD

**Issue by sbddesign** [hsjoberg/blixt-wallet#798]
It appears that Blixt will not successfully scan a BIP21 URI QR code when the `lightning` param is the first param -- only when it is the last param. My understanding of BIP21 is that the parameters do not need to be in a specific order. Phoenix, Muun, Breez, BlueWallet, and Wallet of Satoshi will all scan BIP21s with `lightning` as first or last param.

## How to replicate

1. Scan this QR code with Blixt:
![ln-last](https://user-images.githubusercontent.com/2414177/171962075-eee888e8-6766-4270-9b82-3e9e9bee5bb4.png)
2. It will take you to the Pay Invoice screen
3. Scan this QR code with Blixt:
![ln-first](https://user-images.githubusercontent.com/2414177/171962138-ab66d96c-6589-4360-ba65-993da1cc8e83.png)
4. It will bring up an error message: "QR scan error: Code is not a valid Bitcoin Lightning invoice"
5. Expected result: it should also take me to Pay Invoice screen

**Comment by sbddesign** [comment]
Here's the raw URIs for the above QR codes.

### `lightning` last (works)

```
bitcoin:bc1qylh3u67j673h6y6alv70m0pl2yz53tzhvxgg7u?amount=0.00001&label=sbddesign%3A%20For%20lunch%20Tuesday&message=For%20lunch%20Tuesday&lightning=lnbc10u1p3pj257pp5yztkwjcz5ftl5laxkav23zmzekaw37zk6kmv80pk4xaev5qhtz7qdpdwd3xger9wd5kwm36yprx7u3qd36kucmgyp282etnv3shjcqzpgxqyz5vqsp5usyc4lk9chsfp53kvcnvq456ganh60d89reykdngsmtj6yw3nhvq9qyyssqjcewm5cjwz4a6rfjx77c490yced6pemk0upkxhy89cmm7sct66k8gneanwykzgdrwrfje69h9u5u0w57rrcsysas7gadwmzxc8c6t0spjazup6
```

### `lightning` first (does NOT work)

```
bitcoin:bc1qylh3u67j673h6y6alv70m0pl2yz53tzhvxgg7u?lightning=lnbc10u1p3pj257pp5yztkwjcz5ftl5laxkav23zmzekaw37zk6kmv80pk4xaev5qhtz7qdpdwd3xger9wd5kwm36yprx7u3qd36kucmgyp282etnv3shjcqzpgxqyz5vqsp5usyc4lk9chsfp53kvcnvq456ganh60d89reykdngsmtj6yw3nhvq9qyyssqjcewm5cjwz4a6rfjx77c490yced6pemk0upkxhy89cmm7sct66k8gneanwykzgdrwrfje69h9u5u0w57rrcsysas7gadwmzxc8c6t0spjazup6&amount=0.00001&label=sbddesign%3A%20For%20lunch%20Tuesday&message=For%20lunch%20Tuesday
```

**Comment by hsjoberg** [comment]
Thank you so much for the bug report @sbddesign.
Bug fixed in dc45b8606e24e4f853b4870de4f3147250e302e0.
